plugins/dynamic-plugins-info/app-config.dynamic.yaml

```diff
@@ -0,0 +1,29 @@
+# please keep this in sync with packages/app/src/App.tsx
+dynamicPlugins:
+  frontend:
+    internal.plugin-dynamic-plugins-info:
+      appIcons:
+        - name: pluginsInfoIcon
+          importName: PluginsInfoIcon
+        - name: adminIcon
+          importName: AdminIcon
+      dynamicRoutes:
+        - path: /admin/plugins
+          importName: DynamicPluginsInfoPage
+          menuItem:
+            text: Plugins
+            icon: pluginsInfoIcon
+      mountPoints:
+        - mountPoint: internal.plugins/tab
```

Comment:
Yep, I think this works for me as a sufficiently specific way of mapping where these components need to go now and in the future.  Maybe we can just kind of avoid advertising these new mount points too much :-)